### ISSUE TITLE
TESTNET 5, OSX 10.12.6 Crashing

### ISSUE THREAD

**Issue by 2ge** [syscoin/syscoin#166]
OSX 10.12.6  (Sierra)
```
$ /Applications/Wallets/Syscoin-Qt.app/Contents/MacOS/Syscoin-Qt ; exit;
dyld: Symbol not found: __ZNK5boost16re_detail_10660031cpp_regex_traits_implementationIcE17transform_primaryEPKcS4_
  Referenced from: /Applications/Wallets/Syscoin-Qt.app/Contents/MacOS/../Frameworks/libboost_graph.dylib (which was built for Mac OS X 10.13)
  Expected in: /usr/local/lib/libboost_regex.dylib
 in /Applications/Wallets/Syscoin-Qt.app/Contents/MacOS/../Frameworks/libboost_graph.dylib
Abort trap: 6
logout
Saving session...completed.
Deleting expired sessions...1 completed.
```

**Comment by sidhujag** [comment]
@willyko I think you know why this is on OSX right?

**Comment by willyko** [comment]
Yes...I built is using macdeploy and it's hitting this issue [https://github.com/bitcoin/bitcoin/issues/12126](url)

trying to build it the other way now

There's a workaround where you can just do a `brew install boost`  and it'll install the dependencies and fix that crash (ensure boost version is 1.66)
